java/example_code/dynamodb/src/main/java/com/amazonaws/codesamples/datamodeling/DynamoDBMapperTransactionExample.java

```diff
@@ -0,0 +1,430 @@
+// snippet-sourcedescription:[ ]
+// snippet-service:[dynamodb]
+// snippet-keyword:[Java]
+// snippet-sourcesyntax:[java]
+// snippet-keyword:[Amazon DynamoDB]
+// snippet-keyword:[Code Sample]
+// snippet-keyword:[ ]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[ ]
+// snippet-sourceauthor:[AWS]
```

Comment:
We don't use these tags anymore, so you can remove all of them.

Reply:
Removed.